frontend/src/pages/connectionTypes/manage/ManageConnectionTypeFieldsTableRow.tsx

```diff
@@ -65,13 +72,16 @@ const ManageConnectionTypeFieldsTableRow: React.FC<Props> = ({
   }
 
   return (
-    <Tr id={row.name} data-testid="row">
+    <Tr id={row.id} draggable data-testid="row" {...props}>
+      <Td
+        draggableRow={{
+          id: `draggable-row-${row.id}`,
+        }}
+      />
       <Td dataLabel={columns[0]} data-testid="field-name">
         {row.name}
         <TextContent>
```

Comment:
```suggestion
        <div className="pf-v5-u-color-200">
```

For the section heading above, we should add the same `<Truncate>` for the description.